dev-local-env.yml

```diff
@@ -0,0 +1,44 @@
+---
+# Playbook allowing to deploy a local VM with nested virtualization. This will
+# then be used as the "base" host in order to run CRC + computes from within it,
+# while keeping the host clean.
+
+- name: Bootstrap host with needed bits
+  hosts: "{{ cifmw_lab_host | default('localhost') }}"
+  vars:
+    cifmw_discover_latest_image_base_url: "https://cloud.centos.org/centos/9-stream/x86_64/images"
+  tasks:
+    - name: Ensure we have the needed support
+      ansible.builtin.include_role:
+        name: libvirt_manager
+        tasks_from: virtualization_prerequisites
+
+    - name: Get directories and base packages
+      ansible.builtin.include_role:
+        name: ci_setup
```

Comment:
Seems that with this we are installing "ca-certificates" not needed, and not harmful, but, If we start adding required packages for other envs, we will install more unuseful stuff in developers local envs.

Reply:
We may consider filtering packages if we see we're adding more and more. Though, at this point, I really doubt we'll need more "common/base" packages. More over, ca-certificates is mostly present on FC hosts, especially once we get some softwares such as the JRE and other big things (maybe libreoffice itself is depending on it in some way)